app/views/templates/_news_and_documents.html.erb

```diff
@@ -26,11 +26,15 @@
         if articles.any? %>
           <articles>
 
-            <% articles.each do |a| %>
```

Comment:
A better way to do this would be to set `@article_types = get_article_types` in the controller, then `@article_types` automatically becomes available to use in the view. This is an instance variable in rails, and would help us avoid setting things in the view where possible. I'm not sure how easily we can do this in comfy

We should also try to keep each line wrapped in its own erb tags. I think we can get rid of the concat too as we're not joining anything.